README.md

```diff
@@ -117,10 +117,40 @@ use YieldStudio\NovaGoogleAutocomplete\AddressMetadata;
 use YieldStudio\NovaGoogleAutocomplete\GoogleAutocomplete;
 
 // This autocomplete field will return results that match a business name instead of address.
-// All the same address data is still stored.  
+// All the same address data is still stored.
 GoogleAutocomplete::make('Address')->placeType('establishment');
 ```
 
+### Capturing JSON response
+
+If you want to capture the entire response as a JSON object, you can use the `fromResponse()` helper instead of using `fromValues()`.
+This helper will capture the entire response containing all the requested values as a JSON string.
+
+```php
+// Autocomplete field
+GoogleAutocomplete::make('Location')
+    ->countries('BE')
+    ->withValues(['latitude', 'longitude', 'street_number', 'route', 'locality', 'administrative_area_level_1', 'country', 'postal_code'])
+    ->withMeta(['asJson' => true]),
+
+// Field that will capture de response object
+AddressMetadata::make('Address')
+    ->fromResponse()
```

Comment:
If what you want is not the raw answer from google, but the set of values, I suggest you call this method fromValues() to stay consistent with the other method.

Reply:
`fromValuesAsJson` makes more sense indeed, changed the meta key as well to reflect this.